src/models/queryRangeGroupBy.js

```diff
@@ -0,0 +1,26 @@
+class QueryRangeGroupBy {
+    /**
+     * [constructor]
+     * @param  {String} field
+     * @param  {Array}  ranges mandatory
```

Comment:
as soon as you do not write `Array?` it is a mandatory
could be good also to test if it's an array as you them will map on it (see next remark)